.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+node_js:
+  - 'node'
+before_script:
+  - 'psql -c "CREATE DATABASE db_travis;" -U postgres'
+  - 'psql -c "CREATE USER city WITH PASSWORD ''123'';" -U postgres'
```

Comment:
Are you sure the user city is a good name ? 
And the same for the data base name

Reply:
changed